src/pages/manage/productions/index.js

```diff
@@ -45,22 +45,25 @@ const Index = () => {
     start: currentPageProductions - 1,
     limit: productionsPerPage,
   });
+  const rawProductions = productionsData?.member ?? [];
+
+  useEffect(() => {
+    if (rawProductions.length === 0) return;
+    // make the first production active when new data comes in from the apiCall
+    setActiveProductionId(rawProductions[0].production_id);
+  }, [rawProductions]);
```

Comment:
Was there a situation where the previous solution was breaking things?

Reply:
The previous solution didn't work. The active property was being overwritten as soon as a new activeProductionId was set.

Reply:
Ah yeah, think I got it. Had to read that sentence 10 times though 😆